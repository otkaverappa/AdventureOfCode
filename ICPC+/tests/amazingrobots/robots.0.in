5 4
####
#X.#
#..#
...#
##.#
1
4 3 2 W
4 4
####
#...
#X.#
####
0
