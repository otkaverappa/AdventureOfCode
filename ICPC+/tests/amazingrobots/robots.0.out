8
E
N
E
S
S
S
E
S
